src/bosh_azure_cpi/lib/cloud/azure/storage/blob_manager.rb

```diff
@@ -492,7 +492,7 @@ def apply_retry_policy(retry_data)
       if retry_data[:error].is_a?(OpenSSL::SSL::SSLError) || retry_data[:error].is_a?(OpenSSL::X509::StoreError)
         error_message = retry_data[:error].inspect
 
-        if error_message.include?(Bosh::AzureCloud::Helpers::ERROR_OPENSSL_RESET)
+        if [Bosh::AzureCloud::Helpers::ERROR_OPENSSL_RESET, Bosh::AzureCloud::Helpers:ERROR_OPENSSL_EOF_READ].any? { |error| error_message.include?(error) }
```

Comment:
@anshrupani I believe you are missing an additional colon between `Helpers` and `ERROR_OPENSSL_EOF_READ`.

Reply:
yes you're right, thanks. we're still testing the PR on our environment though